src/cdk/testing/testbed/fake-events/dispatch-events.ts

```diff
@@ -50,6 +51,15 @@ export function dispatchMouseEvent(node: Node, type: string, x = 0, y = 0,
   return dispatchEvent(node, event) as MouseEvent;
 }
 
+/**
+ * Shorthand to dispatch a pointer event on the specified coordinates.
+ * @docs-private
+ */
+export function dispatchPointerEvent(node: Node, type: string, clientX = 0, clientY = 0,
+                                     options?: PointerEventInit): PointerEvent {
+  return dispatchEvent(node, createPointerEvent(type, clientX, clientY, options)) as PointerEvent;
```

Comment:
Don't we need to guard this here in the same way as in `unit-test-element`?

Reply:
I'd say no. If someone calls this method explicitly (which is internally only anyway), then I'd want it to fail if there is no support for pointers. The caller should deal with the support then (as we did in the MDC slider unit tests)